### PR TITLE
User Logout

### PR DESCRIPTION
**Pull request by sethnejame**
# User Logout

Adding feature that allows a user to logout once logged in.  Navbar menu items should be different when user is logged in/logged out.
Pivotal Tracker:
https://www.pivotaltracker.com/story/show/167316810

_As a user,
In order to sign out of my Florum account,
I would like to have a logout function_

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@
     end
 end
 
+Given("I am logged in as {string}") do |name|
+  user = User.find_by(name: name)
+  login_as(user, scope: :user)
```
Comment (oliverochman) [inline features/step_definitions/basic_steps.rb]: can u try writing this is in poetry mode

File: features/user_logout.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Feature: User Log Out
+    As a user,
+    In order to sign out of my account
+    I would like to have a log out function
+
+Background:
```
Comment (oliverochman) [inline features/user_logout.feature]: fix indentation in this file

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: features/user_logout.feature (hunk 2 above)
Comment (sethnejame) [reply features/user_logout.feature]: done!

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (sethnejame) [reply features/step_definitions/basic_steps.rb]: done!

**Review by oliverochman — APPROVED** [review]

**Pull request merged**